manage.py

```diff
@@ -22,5 +39,74 @@ def run_manual(category):
     else:
         print("[-] No methods available.")
 
+@manager.command
+def map_entities():
+    """
+    Run the article crawler for all newspapers.
+    """
+    db = DBManager.get_redis()
+
+    entity_count = dm.get_entity_count()
+    for key in range(1, entity_count + 1):
+        if key % 50 == 0: print("[+] " + str(key) + "th data is processing...")
+        entity = dm.get_entity_by_key(key)
+        db.hset("entity_map", entity.entity_id, key)
+
+@manager.command
+def crawl_article():
+    """
+    Run the article crawler for all newspapers.
+    """
+    db = DBManager.get_redis()
+
+    # TODO(@harrydrippin): Crawl for all newspapers after enough tokens
+    newspaper = "The Verge"
```

Comment:
Please add a TODO for keying newspapers by some ID, not the name of the website.